bin/unlink

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/perl
+
+=begin metadata
+
+Name: unlink
+Description: simpler than rm
+Author: Michael Mikonos
+License:
+
+=end metadata
+
+=cut
+
+use strict;
+
+use File::Basename qw(basename);
+use Getopt::Std qw(getopts);
+
+use constant EX_SUCCESS => 0;
+use constant EX_FAILURE => 1;
+
+my $Program = basename($0);
+
+unless (getopts('')) {
```

Comment:
What's going on here? Are you simply failing if there are any options?

Reply:
I'm curious how this would deal with unlinking a file that starts with a '-'. Can you check that?

Reply:
I call getopt() with no options...
a) so "--" is not treated as a file ("unlink -- FILE" is ok but "unlink --" is incomplete)
b) so arguments like "-f" are treated as an option (unknown option error + usage string is printed)
This is how the GNU version of unlink behaves.

The file "-" is allowed and doesn't require special escaping.
The files "-f" and "--" would normally be given as absolute path or relative to "." ("./-f" and "./--").